packages/info/lib/logger.js

```diff
@@ -13,43 +15,44 @@ const logLevels = { error: 0, warn: 1, info: 2, verbose: 3 };
 class Logger {
   constructor({ name, _workspace }) {
     this.name = name;
-    this.workspace = _workspace;
     this.level = logLevels.info;
+    this.getCleanMessage = (error) =>
+      String(error.stack || error).replace(
+        new RegExp(escapeRegExp(_workspace), 'g'),
+        '.'
+      );
   }
   setLogLevel(level) {
     this.level = level;
   }
-  error(error) {
-    const { level, name, workspace } = this;
+  error(error, ...args) {
+    const { level, name, getCleanMessage } = this;
     const prefix = colorize(`${name}:error`, 'red');
-    const message = String(error.stack || error).replace(
-      new RegExp(escapeRegExp(workspace), 'g'),
-      '.'
-    );
+    const message = getCleanMessage(error);
```

Comment:
Shouldn't we do this for all values in `...args` because any of them could be an error?
We could iterate through the arguments and check for `instanceof Error` or something and then use the `this.getCleanMessage()` method to format the error accordingly?

Reply:
@ZauberNerd I think we decided IRL not to tackle this a this point since we are already mimicking `format`/`inspect` in the current implementation.